src/main/scala-2.12/org/apache/james/gatling/jmap/scenari/JmapInboxHomeLoadingScenario.scala

```diff
@@ -0,0 +1,56 @@
+package org.apache.james.gatling.jmap.scenari
+
+import io.gatling.core.Predef._
+import io.gatling.core.feeder.FeederBuilder
+import io.gatling.core.structure.ScenarioBuilder
+import io.gatling.http.Predef._
+import io.gatling.http.request.builder.HttpRequestBuilder
+import org.apache.james.gatling.jmap._
+
+class JmapInboxHomeLoadingScenario {
+
+  object Keys {
+
+    val inbox = "inboxID"
+
+    val messageIds = "messageIds"
+
+    val messagesDetailList = "messagesDetailList"
+
+  }
+
+
+  private val mailboxListPath = "[0][1].list"
+  private val inboxIdPath = s"$$$mailboxListPath[?(@.role == 'inbox')].id"
+
+  val listAllMailboxesAndSelectFirstOne: HttpRequestBuilder =
+    JmapAuthentication.authenticatedQuery("getMailboxes", "/jmap")
+      .body(StringBody("""[["getMailboxes",{},"#0"]]"""))
+      .check(jsonPath(inboxIdPath).find.saveAs(Keys.inbox))
+
+  val getVacationResponse: HttpRequestBuilder =
+    JmapAuthentication.authenticatedQuery("getVacationResponse", "/jmap")
```

Comment:
this specific call could very well be postponed (and thus removed from this scenario). Please ask Inbox team if they agree to delay display and make Inbox available without that piece of data

Reply:
I asked fabien, and it's ok to remove it.